src/host/polyfills/simpleView.ts

```diff
@@ -217,14 +219,14 @@ export function applyRemoveBreakOnContextMenuItem(content: string): string | nul
 export function applyShowDrawerTabs(content: string): string | null {
     // Appends the Request Blocking tab or Whats New tab in the drawer even if it is not open.
     const pattern = /if\s*\(!view\.isCloseable\(\)\)/;
-    const replacementText = "if(!view.isCloseable()||id==='network.blocked-urls'||id==='release-note')";
+    const replacementText = "if(!view.isCloseable()||id==='network.blocked-urls')";
     return replaceInSourceCode(content, pattern, replacementText);
 }
 
-export function applyPersistDrawerTabs(content: string): string | null {
-    // Removes the close button from the Request blocking and Whats New tab tab by making the tab non-closeable.
+export function applyPersistTabs(content: string): string | null {
+    // Removes the close button from the Request blocking and Welcome tab tab by making the tab non-closeable.
     const pattern = /this\._closeable\s*=\s*closeable;/;
-    const replacementText = "this._closeable= (id==='network.blocked-urls' | id === 'release-note')?false:closeable;";
+    const replacementText = "this._closeable= (id==='network.blocked-urls' || id === 'network' || id === 'welcome')?false:closeable;";
```

Comment:
Yeah, I think making it closeable is the right call. Some options for reopening: bringing in the + button, command menu, adding a custom solution. I'll investigate a bit more

Reply:
I don't think we have enough tools to warrant adding a "+" button unless we want to make everything closeable. I checked and we actually filter out the command menu option to show the panel.

For our command menu patch (`applyCommandMenuPatch`), we filter everything that isn't in the category of `Elements` or `Network`.  We would need to specifically allow the "Panel -> Show Welcome Tab" to get the command menu workflow working.

Reply:
Makes sense. Bringing offline discussion here. I think it would be best if we:
1. Made tab closeable
2. Brought back command menu entry to reopen
3. Added a follow up scenario to create a completely separate welcome page included with the extension available in it's own view. It would show automatically on after updates, and would be available from the extension sidebar

Reply:
Updated PR to do 1 and 2 above